src/less/textbox/stories/icon.stories.js

```diff
@@ -54,3 +54,28 @@ export const dualActionable = () => `
     </button>
 </span>
 `;
+
+export const actionableLarge = () => `
+<span class="textbox textbox--icon-end">
```

Comment:
I think we also need `textbox--large`. This is missing from skin documentation, but is here in the CSS:

https://github.com/eBay/skin/blob/master/dist/textbox/ds6/textbox.css#L153

Reply:
Actually, seems like one or the other. Perhaps, `textbox--large` is deprecated and needs removing...

Reply:
So, I have a feeling these two selectors are legacy, and should be commented as deprecated and to remove in v8:

https://github.com/eBay/skin/blob/master/dist/textbox/ds6/textbox.css#L153